packages/nextjs/src/handlers/logout.ts

```diff
@@ -1,12 +1,12 @@
-import { CookieOptions } from '../types';
+import { CookieOptions } from 'shared/types';
 import type { NextApiRequest, NextApiResponse } from 'next';
-import { setCookies } from '../../shared/utils/cookies';
+import { setCookies } from 'shared/utils/cookies';
```

Comment:
One thing I noticed is that the code imported from `shared` is duplicated in each file that has an import for it. e.g. search for `// ../shared/` in the `nextjs/dist` folder. 

Is there a better way of doing this? E.g. could we tell the bundler to put the shared code into it's own files and import in the other files?

Reply:
Good question, I'll have a look into this as it will be duplicate code in each file otherwise.

Reply:
Ok so the issue here is that because I'm bundling the code it's pulling in all dependencies into the one file. If we truly want to go the bundling route then we would only bundle the intex.ts file since it contains the import/export of all modules. If we want to go the non-bundling route which is what we've been doing in the past then we need to make the `shared` package an actual `npm package`, so we would publish it under `@supabase/auth-helpers-shared-lib` and make it a dependency to `@supabase/auth-helpers-nextjs` (`@supabase/auth-helpers-sveltekit` when its ready.)

Reply:
Okay, I think the bundling is fine for now. The shared code is very light so far.